modules/tinymce/src/plugins/directionality/main/ts/api/Commands.ts

```diff
@@ -9,11 +9,11 @@ import * as Direction from '../core/Direction';
 
 const register = (editor) => {
   editor.addCommand('mceDirectionLTR', () => {
-    Direction.setDir(editor, 'ltr');
+    Direction.setDir(editor, 'ltr' as Direction.Dir);
```

Comment:
We shouldn't need to type cast here

Reply:
This type casting was also hiding a typo, there's normally very few reasons we need to cast